ui/components/ui/typography/typography.js

```diff
@@ -19,15 +20,19 @@ export default function Typography({
   children,
   fontWeight = 'normal',
   align,
+  display,
   boxProps = {},
 }) {
   const computedClassName = classnames(
     'typography',
     className,
```

Comment:
This very much needs to be the last item in this list, thought I'm scared to move it there for how it could effect the rest of the app.

Reply:
Why does it matter what order class names are assigned to a element in? I thought it only mattered what order in the style sheet they are placed.

Reply:
https://codepen.io/darkwing/pen/NWpZWpq

The last `className` wins out when the same property is assigned.  If we provide a `className` to a component, we should safely assume that anything in our className *should*/*must* override.  The way the code in `typography.js` is now, the following override our stuff we put in a CSS class: varient, font-weight, align, color, and display.

That's not good.

Reply:
The className order in the element doesn't matter, switch the red and green classNames and they don't change the element. 


https://user-images.githubusercontent.com/4448075/123104539-a77ae300-d3fc-11eb-80d4-98d7fbf2b979.mov

Reply:
this is a byproduct of the order in which we import component styles, and it's unavoidable without making your import system brittle. The way to get around this is to make the styles for the className you're passing in more specific than the styles that you're attempting to override.

Reply:
Fair enough, not important to be addressed now.